minecraft-server/minecraftd.sh

```diff
@@ -209,17 +212,17 @@ server_stop() {
 			exit 12
 		fi
 
-		if ${SUDO_CMD} screen -S "${IDLE_SESSION_NAME}" -Q select . > /dev/null; then
+		if socket_has_session; then
```

Comment:
This does the wrong thing here: We want to check for whether a session with the `IDLE_SESSION_NAME` is running but `socket_has_session` just checks for `SESSION_NAME`. This is why I proposed to use `tmux -L "${1}" has-session -t"${2:-0:0}" 2>/dev/null` in `socket_has_session` and require the user to always pass the socket name (and the session, pane and window name/number).